specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -3662,6 +3662,10 @@
           "x-nullable": true,
           "readOnly": true
         },
+        "idleTimeBeforeShutdown": {
```

Comment:
>idleTimeBeforeShutdown [](http://example.com/codeflow?start=9&length=22)

 You should use format: date-time here, assuming this format is the same as used elsewhere in this API.

Reply:
That format does not make sense for this property. Here we specify the duration of time in which it will take to go idle: for example 120 minutes would be PT120M. It does not specify a point in time to which it should go idle.

Reply:
OK, that makes sense.